wgpu-hal/src/vulkan/mod.rs

```diff
@@ -234,6 +236,7 @@ struct DeviceShared {
     extension_fns: DeviceExtensionFunctions,
     vendor_id: u32,
     timestamp_period: f32,
+    features: wgt::Features,
```

Comment:
something rubs me the wrong way to see features here, only so that "partially bound" can be checked, but I haven't found a sound way that would be significantly better